winui/AvatarView/getting-started.md

```diff
@@ -1,10 +1,80 @@
 ---
 layout: post
-title: Getting Started with WinUIAvatar View control | Syncfusion
+title: Getting Started with WinUI AvatarView control | Syncfusion
```

Comment:
@MadhumithanJayapathi Learn here all about getting started with the Syncfusion WinUI AvatarView (SfAvatarView) control with its basic features.

Reply:
@Manivannan-E  Changed the summary as mentioned